### DISCUSSION TITLE
Any feedback possible from opening midi editor?

### DISCUSSION THREAD

**Discussion by joeyxsam** [helgoboss/helgobox#1346] [Q&A]
I'd like to be able to open the midi editor in reaper and have that send a message to touchosc.  The purpose of the message would be to automatically change the page on a pager device.  I.e. Page 1 has buttons related to arrange view, page 2 has buttons related to MIDI editor, automatically changing these pages depending on focused context of arrange vs midi editor would be cool.  If possible, what target would I be looking for in realearn to make this happen?

**Comment by helgoboss** [comment]
I can't think of any way to do this with existing targets.
